table.rst

```diff
@@ -0,0 +1,43 @@
+
+Table for astype
+
+
++-------------------+-------------------------+
+|  S1               |  Dtype                  |
++===================+=========================+
+|  int              |  Int8Dtype,Int16Dtype,  |
+|                   |  Int32Dtype,Int54Dtype, |
++-------------------+-------------------------+
+|  str              | StringDtype,str         |
++-------------------+-------------------------+
+|  bytes            | BolleanDtype            |
++-------------------+-------------------------+
+|  float            |Float32Dtype,Float64Dtype|
++-------------------+-------------------------+
+|  complex          |                         |
++-------------------+-------------------------+
+|  Timestamp        |  Timestamp              |
++-------------------+-------------------------+
+|  Timedelta        | Timedelta,TimedeltaIndex|
++-------------------+-------------------------+
+|   Period          | Period, PeriodDtype     |
+|                   | PeriodIndex             |
```

Comment:
Should be just `PeriodDtype(freq)` but I think `Period` can be deleted from `S1` because we have `PeriodSeries`.

However, we will still have an overload of the form:
```python
@overload
def astype(self, dtype: PeriodDtype, other_arguments_etc) -> PeriodSeries: ...
```